conftest.py

```diff
@@ -0,0 +1,4 @@
+import sys, os
```

Comment:
what is this file for?

Reply:
https://docs.pytest.org/en/latest/reference/fixtures.html#conftest-py-sharing-fixtures-across-multiple-files

Reply:
poetry is setup to run all tests in the `tests` folder.  you don't need this.